core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -632,4 +635,5 @@ trait WhiskActionsApi
 
 private case class BlockingInvokeTimeout(activationId: ActivationId) extends TimeoutException
 private case class TooManyActionsInSequence() extends RuntimeException
+private case class NoComponentInSequence() extends RuntimeException
```

Comment:
IllegalArgumentException is what I meant - thanks. That's appropriate. RuntimeExceptions don't fit these.